Fixtravis

10sr opened the pull request:
```
==> Checking out tag v0.1.4
==> sed -ie s|^#!/usr/bin/gawk|#!/usr/bin/env gawk| ext/mwg_pp.awk
==> make
Last 15 lines from /home/travis/.cache/Homebrew/Logs/ble-sh/02.make:
2016-11-07 06:46:05 +0000

make

make[1]: Entering directory `/tmp/ble-sh-20161107-116859-k7k4yz'
mkdir -p out
ext/mwg_pp.awk ble.pp >/dev/null
cp -p term.sh out/term.sh
/usr/bin/env: gawk -f: No such file or directory

make[1]: *** [out/ble.sh] Error 127
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory `/tmp/ble-sh-20161107-116859-k7k4yz'
```

Commit:
Use new linuxbrew repository

Commit:
Remove double install try

Commit:
Fix ble-sh formula

10sr commented:
なんで通らないかな…

akinomyoga commented:
ble.sh が何度も clone されていると思いましたらこちらですね。Shebang `#! ～` の解釈はシステムによって異なります。メジャーなシステムでは
1. 初めの空白までをコマンドのパスとみなし
2. それ以降を一つの引数とみなします(つまり空白で分割されない)

参考: https://cpplover.blogspot.jp/2013/08/shebang.html

従って `#!/usr/bin/env gawk -f` とすると `/usr/bin/env 'gawk -f' ext/mwg_pp.awk ble.pp` になり、最終的に

``` sh:
$ 'gawk -f' ext/mwg_pp.awk ble.pp
```

を呼び出そうとして失敗します。

そもそも ble.sh のビルドが `gawk` のパスに依存するのがいけないので今から修正します。ご迷惑をおかけします。

10sr commented:
わー申し訳ないです！ わざわざありがとうございます :bow: :bow: :bow:

> それ以降を一つの引数とみなします(つまり空白で分割されない)

そうなんですね。。。 OSX では実行できたから別の問題があると思いこんでました。
確かに今 Linux 上で試してみたら再現できました。

Commit:
Merge remote-tracking branch 'origin/master' into fixtravis

Commit:
Add ikiwiki.rb_

Fail because of yaml absence.
I do not know what to do about this...
(brew does not provide yaml perl package)

HOME=/home/me /usr/bin/perl -Iblib/lib   ikiwiki.in --dumpsetup ikiwiki.setup
Can't locate YAML/XS.pm in @INC (you may need to install the YAML::XS module) (@INC contains: . blib/lib /Library/Perl/5.18/darwin-thread-multi-2level /Library/Perl/5.18 /Network/Library/Perl/5.18/darwin-thread-multi-2level /Network/Library/Perl/5.18 /Library/Perl/Updates/5.18.2 /System/Library/Perl/5.18/darwin-thread-multi-2level /System/Library/Perl/5.18 /System/Library/Perl/Extras/5.18/darwin-thread-multi-2level /System/Library/Perl/Extras/5.18) at (eval 36) line 2.

Commit:
Disable sdcv.rb

Commit:
Enable travis cache

Commit:
Use absolute path for brew path

Commit:
Fix make prepare

Commit:
Add vis dependency

Commit:
Uninstall before install in make check

Commit:
Make brew install verbose

Commit:
Disable vis formula

Because link with liblua failed.
The message says liblua should be recompiled with -fPIC option, but I do
not know how to do that.

/home/travis/build/10sr/homebrew-pkg/.brew/bin/gcc-5 -Os -w -pipe -march=native -pipe -Os -fPIE -fstack-protector-all -I/home/travis/build/10sr/homebrew-pkg/.brew/Cellar/libtermkey/0.18/include -D_GNU_SOURCE -I/home/travis/build/10sr/homebrew-pkg/.brew/Cellar/ncurses/6.0_2/include -I/home/travis/build/10sr/homebrew-pkg/.brew/Cellar/ncurses/6.0_2/include/ncursesw   -I/home/travis/build/10sr/homebrew-pkg/.brew/opt/lua/include -DLUA_COMPAT_5_1 -DLUA_COMPAT_5_2 -DLUA_COMPAT_ALL -std=c99 -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700 -DNDEBUG -D_FORTIFY_SOURCE=2 -DVERSION=\"v0.2\" -DCONFIG_LUA=1 -DCONFIG_SELINUX=0 -DCONFIG_ACL=0 *.c -L/home/travis/build/10sr/homebrew-pkg/.brew/lib -Wl,--dynamic-linker=/home/travis/build/10sr/homebrew-pkg/.brew/lib/ld.so -Wl,-rpath,/home/travis/build/10sr/homebrew-pkg/.brew/lib -Wl,-z,now -Wl,-z,relro -Wl,-pie -L/home/travis/build/10sr/homebrew-pkg/.brew/Cellar/libtermkey/0.18/lib -ltermkey -L/home/travis/build/10sr/homebrew-pkg/.brew/Cellar/ncurses/6.0_2/lib -lncursesw   -L/home/travis/build/10sr/homebrew-pkg/.brew/opt/lua/lib -llua -lm -ldl -lc -o vis
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/../lib64/crt1.o: relocation R_X86_64_32S against symbol `__libc_csu_fini' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/Cellar/gcc/5.3.0/bin/../lib/gcc/x86_64-unknown-linux-gnu/5.3.0/crtbegin.o: relocation R_X86_64_32 against hidden symbol `__TMC_END__' can not be used when making a shared object
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lapi.o): relocation R_X86_64_32 against hidden symbol `luaO_nilobject_' can not be used when making a shared object
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(ldebug.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(ldo.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lgc.o): relocation R_X86_64_32 against `.text' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lmem.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lobject.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lparser.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lstate.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(ltable.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(ltm.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lundump.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lvm.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lauxlib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(linit.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lcode.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(llex.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lbaselib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lbitlib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lcorolib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(ldblib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(liolib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lmathlib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(loslib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(lstrlib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(ltablib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: /home/travis/build/10sr/homebrew-pkg/.brew/lib/liblua.a(loadlib.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/home/travis/build/10sr/homebrew-pkg/.brew/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status

Commit:
Make some fix for Makefile

10sr commented:
ひとまずこれで良さそう
